src/spectrecoff-cli/DocumentationUtilities.fs

```diff
@@ -5,37 +5,67 @@ open SpectreCoff.Theming
 
 module Documentation =
     open SpectreCoff
-
-    type Name = Name of string
-    type FunctionSignature = FunctionSignature of string
-
-    type PropertyType = PropertyType of string
-    type DefaultValue = DefaultValue of string
-    type Explanation = Explanation of string
-
-    type DocumentationArtifact = 
-        | FunctionDefinition of Name * FunctionSignature
-        | PropertyDefinition of Name * PropertyType * DefaultValue * Explanation
+    
+    type PropDef = 
+        { Name: string
+          Type: string
+          Explanation: string }
+    
+    type FuncDef = 
+        { Name: string
+          Signature: string }
+    
+    type ValueDef = 
+        { Name: string
+          Type: string
+          DefaultValue: string
+          Explanation: string }
 
     let setDocumentationStyle =
         selectTheme Documentation
         bulletItemPrefix <- "   >> "
         ()
 
-    let private toOutputPayload artifact = 
-        match artifact with
-        | FunctionDefinition (Name n, FunctionSignature s) 
-            -> P $"{n}: {s}"
-        | PropertyDefinition (Name n, PropertyType t, DefaultValue d, Explanation e) 
-            -> Many [ P $"{n}: {t}"; C $"{e} ("; P $"{d}"; C")"]
+    let private leftAlignedColumn header = 
+        column (C header) |> withLayout { defaultColumnLayout with Alignment = Left }
 
-    let print artifacts = 
-        artifacts |> List.map toOutputPayload |> BI
+    let propsOutput (ps: PropDef list) =
+        let columns = 
+            [ leftAlignedColumn "Name"
+              leftAlignedColumn "Type"
+              leftAlignedColumn "Explanation" ]
 
+        ps
+        |> List.map (fun d ->  Payloads [P d.Name; C d.Type; C d.Explanation])
```

Comment:
I am a fan of the implementation, but not exactly of these abbreviations. What does d stand for even?

Reply:
Well, it clearly is a list of property definitions. So d is for definition ;)